core/aggsigdb/memory_internal_test.go

```diff
@@ -15,14 +14,20 @@ import (
 )
 
 func TestDutyExpiration(t *testing.T) {
-	t.Skip("Disabled due to race condition, see https://github.com/ObolNetwork/charon/issues/2546")
+	t.Parallel()
```

Comment:
can remove this as this doesn't bring significant performance improvements

Reply:
Sure, no problem!

Personally, I like to make unit tests parallel not only for the speed but also because they can catch concurrency issues. However, I agree that these tests are fairly simple, and removing the parallelization is reasonable 😃

Reply:
Removed ca5a65632bacf4f1b8ea78de322c8fb1ef210c0a